php/site-utils.php

```diff
@@ -102,6 +109,43 @@ function init_checks() {
 	}
 }
 
+
+function generate_global_docker_compose_yml( Filesystem $fs ) {
+	$img_versions = EE\Utils\get_image_versions();
+
+	$data     = [
+		'services' => [
+			'name'           => 'nginx-proxy',
+			'container_name' => 'ee-nginx-proxy',
+			'image'          => 'easyengine/nginx-proxy:' . $img_versions['easyengine/nginx-proxy'],
+			'restart'        => 'always',
+			'ports'          => [
+				'80:80',
+				'443:443',
+			],
+			'environment'    => [
+				'LOCAL_USER_ID=' . posix_geteuid(),
+				'LOCAL_GROUP_ID=' . posix_getegid(),
+			],
+			'volumes'        => [
+				EE_CONF_ROOT . '/nginx/certs:/etc/nginx/certs',
+				EE_CONF_ROOT . '/nginx/dhparam:/etc/nginx/dhparam',
+				EE_CONF_ROOT . '/nginx/conf.d:/etc/nginx/conf.d',
+				EE_CONF_ROOT . '/nginx/htpasswd:/etc/nginx/htpasswd',
+				EE_CONF_ROOT . '/nginx/vhost.d:/etc/nginx/vhost.d',
+				EE_CONF_ROOT . ':/app/ee4',
+				'/usr/share/nginx/html',
+				'/var/run/docker.sock:/tmp/docker.sock:ro',
+			],
+			'networks'       => [
+				'global-network',
+			],
+		],
+	];
+	$contents = EE\Utils\mustache_render( EE_ROOT . '/templates/global_docker_compose.yml.mustache', $data );
```

Comment:
cross-check the tab-space alignment.

Reply:
Checked. Looks fine to me. Is there a problem :/